eslint.config.js

```diff
@@ -135,18 +135,6 @@ module.exports = [
         ...eslintPluginTestsRecommendedConfig,
         rules: {
             ...eslintPluginTestsRecommendedConfig.rules,
-            "eslint-plugin/test-case-property-ordering": [
-                "error",
-                [
-                    "name",
```

Comment:
`name` isn't included in the new defaults?

Reply:
ops, I forgot to add it, will add it in next major.

I'll revert the change as of now.